.github/workflows/release.yml

```diff
@@ -0,0 +1,143 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Release aissemble
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "The version you want to release aiSSEMBLE on."
+        required: true
+        type: string
+      developmentVersion:
+        description: "The version that dev should be on after this release."
+        required: true
+        type: string
+      releaseBranch:
+        description: "OPTIONAL: The branch to use for running the release."
+        required: false
+        type: string
+      existingVersion:
+        description: "OPTIONAL: The existing aiSSEMBLE version. Defaults to patchVersion-SNAPSHOT"
+        required: false
+        type: string
+      dryRun:
+        description: "OPTIONAL: Used to run the release job as a dry run, skipping the deploy phase"
+        required: false
+        default: false
+        type: boolean
+  push:
+    branches: [ "3-add-release" ]
+
+jobs:
+  build:
+
+    runs-on: arc-runner-set-aissemble
+    env:
+      DOCKER_CONFIG: /home/runner/.docker
+      RELEASE_VERSION: ${{ inputs.releaseVersion || '1.9.0-rc2' }}
+      NEXT_DEVELOPMENT_VERSION: ${{ inputs.developmentVersion || '1.10.0-SNAPSHOT' }}
+      RELEASE_BRANCH: ${{ inputs.releaseBranch }}
+      EXISTING_VERSION: ${{ inputs.existingVersion }}
+      DRY_RUN: ${{ inputs.dryRun || true }}
```

Comment:
C: We should not be including these defaults at all. It just introduces risk/confusion.  If this script is updated/worked on in the future and such mechanisms are helpful, the author can add this stuff in.  However, I would argue that we should be testing by filling in the parameters manually anyway.

Reply:
This is only for testing purposes. This will be deleted before being merged.